docs/env.md

```diff
@@ -237,6 +237,22 @@ The URL for the AWS Cloudfront distribution for the relevant AWS deployment. Thi
 - Required: Yes
 - Default: None
 
+### Crowdin API access related settings
+
+#### CROWDIN_API_KEY
+
+The API access key provided by Crowdin. Only managers can have access to this key.
+
+#### CROWDIN_PROJECT_ID
+
+The unique project identifier provided by Crowdin. Only managers can have access to this setting.
+
+#### CROWDIN_BASE_PATH
+
+The path from where the `crowdin-cli` will work, this path should point to the `src` directory of marsha.
+
+By default its value id `/app/src` and while you are using the container configured in our `docker-compose` file you have no need to change it.
```

Comment:
why not respect the format proposed by @mbenadda for the definition of setting variables?

Reply:
I don't know this format, where can I found it ?

Reply:
Ok I understand. I added the same format as other variable.